### PR TITLE
add new tests for webchat

### PR DESCRIPTION
**Pull request by bhavanibandla**
### Jira link

[HOTT-2102](https://transformuk.atlassian.net/browse/HOTT-2102)

### What?

I have added/removed/altered the following:

- Added new tests to verify webchat related changes on the front end.

### Why?

I am doing this because:

- It's good to have some validation tests for webchat functionality even though it's a static hyperlink on the frontend side.

ACs:

- Tested all changes with respect to the epic HOTT-2102 and its related user stories.

### CODE REVIEW

File: cypress/e2e/WebChat/webChat.cy.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ['', 'xi'].forEach(function(country) {
+    it(`${country} - verify webchat link on help page`, function() {
+      cy.visit(`${country}/help`);
+      cy.contains('Alternatively, you can use our Digital Assistant (opens in new tab) to get help on classifying your goods.');
+      cy.get('#webchat-link > a').should('have.attr', 'href', 'https://www.tax.service.gov.uk/ask-hmrc/chat/trade-tariff');
+    });
+    it(`${country} - Search and verify webchat message and link is displayed`, function() {
+      cy.visit(`${country}/find_commodity`);
+      cy.searchForCommodity('xxxx');
+      cy.contains('Getting help from HMRC if you need to find a commodity code');
```
Comment (willfish) [inline cypress/e2e/WebChat/webChat.cy.js:11]: Out of interest, does this run fine on both staging and dev?

**Review by willfish — APPROVED** [review]
LGTM

I do think testing static content will make our life harder later

File: cypress/e2e/WebChat/webChat.cy.js (hunk 1 above)
Comment (bhavanibandla) [reply cypress/e2e/WebChat/webChat.cy.js:11]: yes, it does!

File: cypress/e2e/WebChat/webChat.cy.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+describe('Verify Webchat URL link and access on various pages', function() {
+  ['', 'xi'].forEach(function(country) {
+    it(`${country} - verify webchat link on help page`, function() {
+      cy.visit(`${country}/help`);
+      cy.contains('Alternatively, you can use our Digital Assistant (opens in new tab) to get help on classifying your goods.');
```
[the remark this answers is not in the document]
Comment (bhavanibandla) [reply cypress/e2e/WebChat/webChat.cy.js:5]: In that case, we can use just the 'Digital Assistant (opens in new tab)' text to verify.

File: cypress/e2e/WebChat/webChat.cy.js (hunk 1 above)
Comment (willfish) [reply cypress/e2e/WebChat/webChat.cy.js:11]: Perfect!

**Pull request merged**